nixos/maintainers/scripts/oci/upload-image.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh -eu
```

Comment:
I would prefer using bash explicitly because /bin/sh is bash on nixpkgs but dash/ash/etc on other systems.